### PR TITLE
feat: lending handle hard cap reached

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

... and fix wrong condition for savers while at it, that made the Twitter CTA not display when it should

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

> What protocols, transaction types or contract interactions might be affected by this PR?

Low, as this *disables* lending and adds info div / Twitter CTA but could disable loans when actually not halted

## Testing

- Both the ETH and BTC pools are currently showing a hard cap filled CTA
- It is impossible to continue with lending deposits when halted

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

Screenshot 2024-02-28 at 11 32 08
Screenshot 2024-02-28 at 11 32 26
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* **#6321** <a href="https://app.graphite.dev/github/pr/shapeshift/web/6321?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/6321?utm_source=stack-comment).

File: src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx [hunk 1]
```diff
@@ -304,23 +304,21 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
...
-                  isExternal
-                  mt={4}
-                  colorScheme='twitter'
-                  rightIcon={faTwitterIcon}
-                >
-                  @THORChain
-                </Button>
-              </>
-            )}
+            <>
```
Comment (gomesalexandre) [inline src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx:307]: This was obviously wrong

File: src/pages/Lending/Pool/components/PoolInfo.tsx [hunk 2]
```diff
@@ -60,6 +75,58 @@ export const PoolInfo = ({ poolAssetId }: PoolInfoProps) => {
...
+            value={currentCapFillPercentage}
+            size='sm'
+            borderRadius='md'
+            colorScheme={bnOrZero(currentCapFillPercentage).lt(100) ? 'green' : 'red'}
+          />
+        )}
+      </Flex>
+    )
+  }, [alertBg, asset, poolData, translate])
+
```
Comment (woodenfurniture) [inline src/pages/Lending/Pool/components/PoolInfo.tsx:129]: nitpicky but is there a reason this guy isnt a separate component?

**Comment by woodenfurniture** [comment]
# Testing notes

✅ Twitter button works

image

---

✅ Asks me to harass thorchain to raise caps when deposits halted with vault cap full

image

---

unable to test non-full vault case since btc and eth are both halted

**Review by woodenfurniture — APPROVED** [review]
lgtm

File: src/pages/Lending/Pool/components/PoolInfo.tsx (hunk 2 above)
Comment (gomesalexandre) [reply src/pages/Lending/Pool/components/PoolInfo.tsx:129]: Not really, I guess just not generic/reused enough to be moved to its own component?

**Pull request merged**